### PR TITLE
Know Your Rights: Static Screens

### PR DESCRIPTION
**Pull request by khaliunn**
- Moved "Home Screen" from ExampleTabScreen to KnowYourRightsScreen
- Created placeholder Alerts and Resources screens
- Configured bottom tab to resemble mid-fidelity frames as closely as possible
- Set up stack navigation for Know Your Rights screens (the header needs to be configured)
- Created reusable components across Know Your Rights screens
- Filled Know Your Rights screens with sample components (couldn't get the specifics because of Figma outage)
Screenshot_20200616-211313
Screenshot_20200616-211326
Screenshot_20200616-211332
Screenshot_20200616-211337
Screenshot_20200616-211401
Screenshot_20200616-211404

### CODE REVIEW

**Comment by kikichan513** [comment]
eslint failing 
```
/action/lib/entrypoint.sh: 5: cd: can't cd to optional-sub-dir
source root not found
yarn install v1.22.4
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.2.13: The platform "linux" is incompatible with this module.
info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
```

File: js/screens/knowyourrights/components.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                <Text style={styles.tipsText}>{tip}</Text>
+            </View>
+        )
+    })
+
+    return (
+        <View>
+            <Text style={styles.tipsTitleText} >Tips</Text>
+            <View style={styles.actionContainer}>
+                {tipsList}
```
Comment (JacobJaffe) [inline js/screens/knowyourrights/components.js:75]: Tryout using a `FlatList` for this kind of map data!

File: js/screens/styles.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import { StyleSheet } from 'react-native'
```
Comment (JacobJaffe) [inline js/screens/styles.js:1]: Can this go under `js/styles/` ?

File: js/screens/knowyourrights/DrivingScreen.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { ScrollView } from 'react-native-gesture-handler';
+
+import styles from '../styles'
+import { RightsTitle, RightsWarning, ActionBullets, ActionItems, Tips } from './components'
+
+export default Driving = () => {
+    return (
+        <ScrollView contentContainerStyle={styles.container}>
```
Comment (JacobJaffe) [inline js/screens/knowyourrights/DrivingScreen.js:9]: Looks like all these screens have the same template -- try out making a file `KnowYourRights.js` that takes in these strings and consolidates the forming

**Pull request merged**